package.json

```diff
@@ -35,6 +36,7 @@
     "bluebird": "~3.5.1",
     "cache-manager": "~2.8.0",
     "cache-manager-redis": "0.5.0",
+    "classnames": "^2.2.5",
```

Comment:
we always use `~`, not `^`. this is related to other dependencies introduced by this PR too